zips/zip-0253.md

```diff
@@ -0,0 +1,80 @@
+
+    ZIP: 253
+    Title: Deployment of the NU6 Network Upgrade
+    Owners: Arya <arya@zfnd.org>
+    Status: Draft
+    Category: Consensus / Network
+    Created: 2024-07-17
+    License: MIT
+    Discussions-To: <https://github.com/zcash/zips/issues/806>
+
+# Terminology
+
+The key word "MUST" in this document are to be interpreted as described in
+BCP 14 [^BCP14] when, and only when, they appear in all capitals.
+
+The term "network upgrade" in this document is to be interpreted as described in ZIP 200 [^zip-200].
+
+The terms "Testnet" and "Mainnet" are to be interpreted as described in
+section 3.12 of the Zcash Protocol Specification  [^protocol-networks].
+
+# Abstract
+
+This proposal defines the deployment of the NU6 network upgrade.
+
+# Specification
+
+## NU6 deployment
+
+<!-- TODO: Update these references once ZIP numbers are assigned to the draft ZIPs -->
+
+The primary sources of information about NU6 consensus protocol changes are:
+
+* The Zcash Protocol Specification [^protocol].
+* Network Upgrade Mechanism [^zip-200].
+* Lockbox Funding Streams [^lockbox-funding-streams].
+* Block Reward Allocation for Non-Direct Development Funding [^block-reward-allocation-for-non-direct-dev-fund].
+* Blocks should balance exactly [^coinbase-should-balance-exactly].
```

Comment:
We always use the name of the file as the reference link (in part to make it easy to find-and-replace later):
```suggestion
* Network Upgrade Mechanism [^zip-0200].
* Lockbox Funding Streams [^zip-2001].
* Block Reward Allocation for Non-Direct Development Funding [^draft-nuttycom-funding-allocation].
* Blocks should balance exactly [^zip-0236].
```

Please similarly fix all other `zip-` reference links.

Reply:
Applied in https://github.com/zcash/zips/pull/878/commits/bb4e171b0174de2f609f5b1952edb4f6c4b3c8ed. That's much nicer, thank you.